### PR TITLE
Gladys - Fix docker-compose.yml

### PR DESCRIPTION
**Pull request by cicoub13**
Here is my working version of Gladys (tested on RPI4)

### CODE REVIEW

File: Apps/gladys/docker-compose.yml [hunk 1]
```diff
@@ -11,50 +11,19 @@ services:
...
+    container_name: gladys
+    cgroup: host
     environment: # Environment variables for the container
       NODE_ENV: production # Sets the environment to production
       SQLITE_FILE_PATH: /var/lib/gladysassistant/gladys-production.db # Path to the SQLite database
+      TZ: Europe/Paris # Timezone
+      SERVER_PORT: "1080" # Server port for UI
     volumes: # Mount points from the host to the container
       - /var/run/docker.sock:/var/run/docker.sock # Docker socket file for Docker within Docker
-      - /DATA/AppData/$AppID/data:/var/lib/gladysassistant # Persistent storage for Gladys data
```
Comment (dragonfire1119) [inline Apps/gladys/docker-compose.yml:19]: In CasaOS you put app data in /DATA/AppData/*

File: Apps/gladys/docker-compose.yml [hunk 2]
```diff
@@ -11,50 +11,19 @@ services:
...
+      - /var/lib/gladysassistant:/var/lib/gladysassistant # Persistent storage for Gladys data
       - /dev:/dev # Access to host devices
-    ports: # Port mappings from the host to the container
-      - "1080:80" # Maps port 80 in the container to port 1080 on the host
-      - "10443:443" # Maps port 443 in the container to port 10443 on the host
-
-    x-casaos: # CasaOS specific configuration
```
Comment (dragonfire1119) [inline Apps/gladys/docker-compose.yml:25]: The x-casaos needs to stay with describing the envs and volumes.

**Review by dragonfire1119 — CHANGES_REQUESTED** [review]

File: Apps/gladys/docker-compose.yml (hunk 1 above)
Comment (cicoub13) [reply Apps/gladys/docker-compose.yml:19]: Fixed

File: Apps/gladys/docker-compose.yml (hunk 2 above)
Comment (cicoub13) [reply Apps/gladys/docker-compose.yml:25]: Fixed

File: Apps/gladys/docker-compose.yml [hunk 3]
```diff
@@ -66,22 +66,26 @@ x-casaos:
...
+  thumbnail: https://gladysassistant.com/img/home/main_screenshot/main_screenshot_en_j5czyj_c_scale,w_2526.png
+  screenshot_link:
+    - https://gladysassistant.com/img/home/main_screenshot/main_screenshot_en_j5czyj_c_scale,w_2526.png
   title:
     # Title in English
-    en_us: Gladys Assistant
+    en_us: "Gladys Assistant"
   # Application category
-  category: BigBearCasaOS
+  category: "Home Automation"
```
Comment (dragonfire1119) [inline Apps/gladys/docker-compose.yml:89]: This needs to stay BigBearCasaOS.

File: Apps/gladys/docker-compose.yml [hunk 4]
```diff
@@ -66,22 +66,26 @@ x-casaos:
...
+    en_us: "Gladys Assistant is a modern, privacy-first & open-source home automation software that runs anywhere."
+    fr_fr: "Gladys Assistant est un logiciel de domotique moderne et respectueux de la vie privée."
   tagline:
     # Short description or tagline in English
-    en_us: A privacy-first, open-source home assistant
+    en_us: "A privacy-first, open-source home assistant."
+    fr_fr: "Le logiciel de domotique open-source qui respecte votre vie privée."
   # Developer's name or identifier
-  developer: "GladysAssistant"
+  developer: "Gladys Assistant"
```
Comment (dragonfire1119) [inline Apps/gladys/docker-compose.yml:76]: This needs to stay like the org on github: GladysAssistant

**Review by dragonfire1119 — CHANGES_REQUESTED** [review]
Looking good.

File: Apps/gladys/docker-compose.yml (hunk 4 above)
Comment (cicoub13) [reply Apps/gladys/docker-compose.yml]: Fixed

File: Apps/gladys/docker-compose.yml (hunk 3 above)
Comment (cicoub13) [reply Apps/gladys/docker-compose.yml]: Fixed

**Pull request merged**